inbm/dispatcher-agent/dispatcher/fota/guid.py

```diff
@@ -28,7 +27,7 @@ def extract_guids(fw_tool: str, types: list[str]) -> list[str]:
         raise FotaError("Firmware Update Aborted: failed to list GUIDs: {}".format(str(err)))
```

Comment:
<h3>$\color{#FFD700}Coding Style$</h3>

The error message in `raise FotaError("Firmware Update Aborted: failed to list GUIDs: {}".format(str(err)))` could be improved by using f-strings for consistency and readability.